DayPicker/tests/DayPicker-specs.js

```diff
@@ -78,26 +79,17 @@ describe('DayPicker', () => {
 		expect(label).toBe(expected);
 	});
 
-	test('should return `Every Day` when all selected', () => {
-		const selected = [0, 1, 2, 3, 4, 5, 6];
-		const label = getSelectedDayString(selected);
-		const expected = 'Every Day';
-		expect(label).toBe(expected);
-	});
-
```

Comment:
Why is this test deleted?

Reply:
I think it was hiding from me. I added it back. Thanks!